packages/unison/unison.2.53.8/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+maintainer: "unison-hackers@lists.seas.upenn.edu"
+authors: [
+  "Trevor Jim"
+  "Benjamin C. Pierce"
+  "Jérôme Vouillon"
+]
+license: "GPL-3.0-or-later"
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "https://github.com/bcpierce00/unison/issues"
+dev-repo: "git://github.com/bcpierce00/unison.git"
+patches: ["unison.patch"]
+build: [
+  [make "NATIVE=%{ocaml:native}%" "-j" jobs] {!host-system-msvc:installed}
+  ["nmake" "NATIVE=%{ocaml:native}%"] {host-system-msvc:installed}
+]
+install: [
+  [make "install" "PREFIX=%{prefix}%"] {os != "win32"}
+  ["cmd" "/c" "copy" "src\\unison*.exe" "%{prefix}%\\bin"] {os = "win32"}
+]
+depends: [
+  "ocaml" {>= "4.08"}
+]
+depopts: [
+  "lablgtk3" {>= "3.1.0"}
+  "ocamlfind"
```

Comment:
Note to self: this is a surprising depopt, rather than a `{build}`? (given that unison-gui requires it?)